news_items/032_course_permissions/index.html

```diff
@@ -0,0 +1,259 @@
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Close issues</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Edit question code and JSON files</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Sync from GitHub</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Edit course staff permissions</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+</table>
+
+<h3 class="mt-5">Student data access</h3>
+
+<p>
+  Student data access permissions apply for individual course
+  instances, so instructors and TAs can be given access to student
+  data in Fall semester but not Spring semester, for example. These
+  permissions can be mixed and matched with the above course content
+  permissions, so you can separately control access to course content
+  from access to student data. The new student data permissions are:
+</p>
+
+<table class="table table-striped table-sm border mx-auto" style="max-width: 30em">
+  <tr>
+    <th></th>
+    <th class="text-center" scope="col" colspan="5">Student data access</th>
+  </tr>
+  <tr>
+    <th scope="col">Action</th>
+    <th class="text-center" scope="col">None</th>
+    <th class="text-center" scope="col">Viewer</th>
+    <th class="text-center" scope="col">Editor</th>
+  </tr>
+  <tr>
+    <td>View individual student scores</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Download gradebook data</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Manually grade student submissions</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+  <tr>
+    <td>Edit individual student scores</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&centerdot;</td>
+    <td class="text-center">&check;</td>
+  </tr>
+</table>
```

Comment:
Maybe add: "Change time limits" and "Close assessments" as options?

Reply:
Done.